mu/wheels/__init__.py

```diff
@@ -19,9 +19,7 @@
 mode_packages = [
     ("pgzero", "pgzero>=1.2.1"),
     ("flask", "flask==1.1.2"),
-    ("serial", "pyserial>=3.0"),
     ("qtconsole", "qtconsole==4.7.4"),

```

Comment:
Tried to remove this one as well, but get the following error when launching the Mu REPL:

```
Traceback (most recent call last):
  File "/Users/microbit-carlos/workspace/mine/mu-upstream/mu/modes/python3.py", line 109, in start_kernel
    self.repl_kernel_manager.start_kernel()
  File "/Users/microbit-carlos/workspace/mine/mu-upstream/mu/modes/python3.py", line 42, in start_kernel
    kernel_cmd, kw = self.pre_start_kernel(**kw)
  File "/Users/microbit-carlos/workspace/mine/mu-upstream/.venv/lib/python3.6/site-packages/jupyter_client/manager.py", line 292, in pre_start_kernel
    kernel_cmd = self.format_kernel_cmd(extra_arguments=extra_arguments)
  File "/Users/microbit-carlos/workspace/mine/mu-upstream/.venv/lib/python3.6/site-packages/jupyter_client/manager.py", line 210, in format_kernel_cmd
    cmd = self.kernel_spec.argv + extra_arguments
  File "/Users/microbit-carlos/workspace/mine/mu-upstream/.venv/lib/python3.6/site-packages/jupyter_client/manager.py", line 114, in kernel_spec
    self._kernel_spec = self.kernel_spec_manager.get_kernel_spec(self.kernel_name)
  File "/Users/microbit-carlos/workspace/mine/mu-upstream/.venv/lib/python3.6/site-packages/jupyter_client/kernelspec.py", line 230, in get_kernel_spec
    raise NoSuchKernel(kernel_name)
jupyter_client.kernelspec.NoSuchKernel: No such kernel named mu_venv-36-20210412-001637
```

QtConsole has more dependencies than needed for the venv, so maybe we can replace it with something else, but this is simpler for now.